pubsub.go

```diff
@@ -16,18 +19,23 @@ type PubSub struct {
 }
 
 type settings struct {
-	publish PublishSettings
-	receive ReceiveSettings
+	publish      PublishSettings
+	receive      ReceiveSettings
+	subscription SubscriptionSettings
+	topic        TopicSettings
 }
 
 // Config provides the information needed to securely connect to Google Cloud's PubSub
 // and to configure any publishing and subscription options.
 type Config struct {
-	ProjectID              string
-	IsLocal                bool
-	ServiceAccountFilePath string
-	PublishSettings        PublishSettings
-	ReceiveSettings        ReceiveSettings
+	IsLocal                         bool
+	ProjectID                       string
+	PublishSettings                 PublishSettings
+	ReceiveSettings                 ReceiveSettings
+	ServiceAccountFilePath          string
+	SubscriptionEnableOrdering      bool
```

Comment:
I would just list `SubscriptionSettings` as the field since that struct has the enable ordering and retain acked messages settings. Similar to how the `Config` takes `PublishSettings` instead of individual fields that make up `PublishSettings`